src/lib/github.js

```diff
@@ -115,10 +115,13 @@ export const getApprovalStatus = async (pullNumber) => {
     pull_number: pullNumber,
   });
 
+  let reviewers = new Set();
   let changesRequestedCount = 0;
   let approvalCount = 0;
 
-  reviewsData.forEach(({ state }) => {
+  reviewsData.reverse().forEach(({ state, user }) => {
+    if (reviewers.has(user.login)) return;
+    reviewers.add(user.login);
```

Comment:
I just realized this will introduce some new bug.
for example, if we have a list of reviews like below
```
[ userA: approve, userB: approve, userA: comment ]
```

Reply:
Oh, indeed. I guess we should only add the user to the reviewers set if the state was either "changes requested" or "approved".

Reply:
Correct, something like this

![image](https://user-images.githubusercontent.com/681150/113926995-8538d800-97b2-11eb-899a-5bbab3c9b254.png)

Reply:
@nafur I have merged master into this branch, and the test failed. Can you add a quick fix?
It would be great if you can some test cases like
- `[ userA: approve, userB: approve, userA: comment ]` 
- `[ userA: approve, userA: approve ]`

Reply:
Done.

Reply:
Great, thanks Nafur